packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/DdSessionReplayImplementationTest.kt

```diff
@@ -56,6 +55,23 @@ internal class DdSessionReplayImplementationTest {
     @Mock
     lateinit var mockUiManagerModule: UIManagerModule
 
+    private val imagePrivacyMap = mapOf(
+        "MASK_ALL" to ImagePrivacy.MASK_ALL,
+        "MASK_NON_BUNDLED_ONLY" to ImagePrivacy.MASK_LARGE_ONLY,
```

Comment:
why key is different from the enum name? upd: ah, I guess it is because on iOS it is `.maskNonBundledOnly`?

Reply:
Exactly!